src/index.js

```diff
@@ -42,6 +43,7 @@ class Thermometer extends Component {
       height: this.props.height || 200, //default 200
       valstr: () => this.options.value + this.options.format,
       percent: () => this.options.value / this.options.max * 100,
+      reverseGradient: this.props.reverseGradient === 'true' ? true : false, // default false
```

Comment:
If you change reverseGradient to boolean you can change this to: 

this.props.reverseGradient || false

Reply:
:+1: